content/python/concepts/built-in-functions/terms/filter/filter.md

```diff
@@ -0,0 +1,45 @@
+---
+Title: 'filter()'
+Description: 'A built-in function that applies a function to each item in an iterable and returns the values that are true.'
+Subjects:
+  - 'Computer Science'
+  - 'Data Science'
+Tags:
+  - 'Functions'
+  - 'Methods'
+  - 'Strings'
+CatalogContent:
+  - 'learn-python-3'
+  - 'paths/computer-science'
+  - 'paths/data-science'
+---
+
+'A built-in function that applies a function to each item in an interable and returns the values that are true. The output of `filter()` is a filter object, in the examples below `list()` is applied to the filter expressions to return a new list.'
+
+## Syntax
+
+```py
+filter(func_name, iterable)
+```
+
```

Comment:
Let's add 2-3 sentences here (or a list) that describes what each function argument is and does. 😄